files/najaxjs/update.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packageManager": "github",
+  "name": "najaxjs",
+  "repo": "any-js/najaxjs",
+  "files": {
+    "basePath": "dist/",
+    "include": [
+      "najax.js", "najax.min.js",
```

Comment:
Can you just leave the min files?
You can use this rule `*.min.*` to add them all